sdk/containerservice/Azure.ResourceManager.ContainerService/api/Azure.ResourceManager.ContainerService.netstandard2.0.cs

```diff
@@ -1054,6 +1091,12 @@ public partial class ManagedClusterNatGatewayProfile
         public int? IdleTimeoutInMinutes { get { throw null; } set { } }
         public int? ManagedOutboundIPCount { get { throw null; } set { } }
     }
+    public partial class ManagedClusterOidcIssuerProfile
+    {
+        public ManagedClusterOidcIssuerProfile() { }
+        public bool? Enabled { get { throw null; } set { } }
+        public string IssuerURL { get { throw null; } }
```

Comment:
Is the type Uri?

Reply:
I think it should be, but cannot find samples to prove it, therefore I renamed it to `IssuerUriInfo`